cosim/elaborate/src/DUT.scala

```diff
@@ -0,0 +1,183 @@
+package cosim.elaborate
+
+import chisel3._
+import chisel3.util.Decoupled
+import freechips.rocketchip.diplomacy.{AddressSet, BundleBridgeSource, InModuleBody, LazyModule, RegionType, SimpleLazyModule, TransferSizes}
+import freechips.rocketchip.interrupts.{IntSinkNode, IntSinkPortSimple, IntSourceNode, IntSourcePortSimple}
+import freechips.rocketchip.subsystem._
+import freechips.rocketchip.tilelink.{TLManagerNode, TLSlaveParameters, TLSlavePortParameters}
+import org.chipsalliance.cde.config.Parameters
+import freechips.rocketchip.tile.{NMI, PriorityMuxHartIdFromSeq, RocketTile}
+import org.chipsalliance.tilelink.bundle._
+import org.chipsalliance.cde.config.{Config, Field}
+import freechips.rocketchip.diplomacy._
+
+class DUT(p: Parameters) extends Module {
+  implicit val implicitP = p
+  val tileParams = p(RocketTileParamsKey)
+  val ldut = LazyModule(new SimpleLazyModule with BindingScope {
+    implicit val implicitP = p
+    val rocketTile = LazyModule(new RocketTile(tileParams, RocketCrossingParams(), PriorityMuxHartIdFromSeq(Seq(tileParams))))
+    val masterNode = TLManagerNode(Seq(TLSlavePortParameters.v1(
+      Seq(TLSlaveParameters.v1(
+        address = List(AddressSet(0x0, 0xffffffffL)),
+        regionType = RegionType.UNCACHED,
+        executable = true,
+        supportsGet = TransferSizes(1, 64),
+        supportsAcquireT = TransferSizes(1, 64),
+        supportsAcquireB = TransferSizes(1, 64),
+        supportsPutPartial = TransferSizes(1, 64),
+        supportsPutFull = TransferSizes(1, 64),
+        supportsLogical = TransferSizes(1, 64),
+        supportsArithmetic = TransferSizes(1, 64),
+        fifoId = Some(0))),
+      beatBytes = 8,
+      endSinkId = 4,
+      minLatency = 1
+    )))
```

Comment:
This node should be automatically generated by memory node in `rocketTile` rather than hard code.